ec/resources/waf/access_rule.go

```diff
@@ -427,18 +427,8 @@ func ResourceAccessRuleCreate(ctx context.Context, d *schema.ResourceData, m int
 		}
 	}
 
-	log.Printf("[DEBUG] Allowed HTTP Methods: %+v\n", accessRule.AllowedHTTPMethods)
-	log.Printf("[DEBUG] Allowed Request Content Types: %+v\n", accessRule.AllowedRequestContentTypes)
-	log.Printf("[DEBUG] Disallowed Headers: %+v\n", accessRule.DisallowedHeaders)
-	log.Printf("[DEBUG] Disallowed Extensions: %+v\n", accessRule.DisallowedExtensions)
-	log.Printf("[DEBUG] ASN: %+v\n", accessRule.ASNAccessControls)
-	log.Printf("[DEBUG] Cookie: %+v\n", accessRule.CookieAccessControls)
-	log.Printf("[DEBUG] Country: %+v\n", accessRule.CountryAccessControls)
-	log.Printf("[DEBUG] IP: %+v\n", accessRule.IPAccessControls)
-	log.Printf("[DEBUG] Referer: %+v\n", accessRule.RefererAccessControls)
-	log.Printf("[DEBUG] URL: %+v\n", accessRule.URLAccessControls)
-	log.Printf("[DEBUG] User Agent: %+v\n", accessRule.UserAgentAccessControls)
-
+	helper.LogInstanceAsPrettyJsonToFile("[DEBUG] ACCESSRULE", accessRule, "access-rule.log")
```

Comment:
Unfortunately, printing a Go object with Pointer properties will result in memory addresses being printed... this is why I printed everything manually using many log.Printf statements. It doesn't look pretty but I think we need to go back to line-by-line property printing.

Reply:
LogInstanceAsPreetyJsonToFile is not just printing point address.
If you look at the implementation, it converts it as Json.
However, I don't think we need to have a separate log file I will remove it.

Reply:
@changgyu-oh I see, cool!